annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/helpers/FieldSetterNameGenerator.java

```diff
@@ -0,0 +1,12 @@
+package com.exactpro.epfast.annotation.processing.helpers;
+
+import java.util.Base64;
+
+public class FieldSetterNameGenerator {
+    private static Base64.Encoder encoder = Base64.getEncoder().withoutPadding();
+
+    public static String generateClassName(String fastTypeName) {
+        String hexString = "A" + encoder.encodeToString(fastTypeName.getBytes());
```

Comment:
specify UTF-8 encoding for getBytes explicitly. You don't need hexString variable. You can return computation result w/o storing it first.